pom.xml

```diff
@@ -113,7 +113,7 @@
         <retrofit.version>2.5.0</retrofit.version>
         <saaj.impl.version>1.5.3</saaj.impl.version>
         <scala-2.11.version>2.11.12</scala-2.11.version><!-- Spark -->
-        <smallrye.reactive.messaging.camel.version>2.9.0</smallrye.reactive.messaging.camel.version> <!-- keep in sync with Quarkus SmallRye Reactive Messaging -->
+        <smallrye.reactive.messaging.camel.version>3.3.2</smallrye.reactive.messaging.camel.version> <!-- keep in sync with Quarkus SmallRye Reactive Messaging -->
```

Comment:
Then, something like this could work:

```suggestion
        <smallrye.reactive.messaging.camel.version>3.3.2</smallrye.reactive.messaging.camel.version><!-- @sync io.quarkus:quarkus-bom:${quarkus.version} dep:io.smallrye.reactive:smallrye-reactive-messaging-provider -->
```

Note that the rule does not reference the version property, because the version property is not available in the flattened BOM they publish to Central. So the tool rather picks the version from a dependency that is known to use the desired version property.

Reply:
Thanks, applied.